src/python/library/tritonclient/grpc/__init__.py

```diff
@@ -198,7 +203,8 @@ def __init__(self,
                  root_certificates=None,
                  private_key=None,
                  certificate_chain=None,
-                 keepalive_options=None):
+                 keepalive_options=None,
+                 creds=None,):
```

Comment:
Can you move `creds` above `keepalive_options` here as well?  And also remove the trailing comma.

Reply:
Done